### PR TITLE
Markdown conversion for `pt-BR` - Replace - Learning Area section :warning:  Do not squash :warning:

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of https://github.com/mdn/translated-content/pull/8466
https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-15t04_25_01-866z-md

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

**Comment by SphinxKnight** [comment]
Conversion report updated after rebasing this work up onto upstream to fix conflicts https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-26t11_12_11-415z-md

File: files/pt-br/learn/forms/your_first_form/index.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+```html
+<form action="/pagina-processa-dados-do-form" method="post">
+
+</form>
+```
+
+Este elemento define um formulário. É um elemento de container como um elemento {{HTMLElement ("div")}} ou {{HTMLElement ("p")}} , mas ele também suporta alguns atributos específicos para configurar a forma como o formulário se comporta. Todos os seus atributos são opcionais, mas é considerada a melhor prática sempre definir pelo menos o atributo `action `e o atributo `method`.
+
+- O atributo \_**action** \_define o local (uma URL) em que os dados recolhidos do formulário devem ser enviados.
+- O atributo _**method**_ define qual o método HTTP para enviar os dados (ele pode ser "**GET**" ou "**POST**" (veja as diferenças [aqui](http://www.comocriarsites.com/html/como-funciona-os-metodos-get-e-post-diferencas/)).
```
Comment (josielrocha) [inline files/pt-br/learn/forms/your_first_form/index.md:54]:
```suggestion
- O atributo _**action**_ define o local (uma URL) em que os dados recolhidos do formulário devem ser enviados.
- O atributo _**method**_ define qual o método HTTP para enviar os dados (ele pode ser "**GET**" ou "**POST**" (veja as diferenças [aqui](http://www.comocriarsites.com/html/como-funciona-os-metodos-get-e-post-diferencas/)).
```

File: files/pt-br/learn/forms/your_first_form/index.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+Por último, mas não menos importante, observe a sintaxe de `<input /> `e `<textarea> </ textarea>.` Esta é uma das esquisitices do HTML. A tag** `<input />` **é um elemento que se auto-fecha, o que significa que se você quiser encerrar formalmente o elemento, você tem que adicionar uma barra "**/**" no final do próprio elemento e não uma tag de fechamento. No entanto, o tipo {{HTMLElement ("textarea")}} não é um elemento de auto-fechamento, então você tem que fechá-lo com a tag final adequada. Isso tem um impacto sobre um recurso específico de formulários HTML: a maneira como você define o valor padrão. Para definir o valor padrão de um elemento {{HTMLElement ("input")}} você tem que usar o atributo `value `como este:
```
Comment (josielrocha) [inline files/pt-br/learn/forms/your_first_form/index.md:85]:
```suggestion
Por último, mas não menos importante, observe a sintaxe de `<input /> `e `<textarea> </ textarea>.` Esta é uma das esquisitices do HTML. A tag **`<input />`** é um elemento que se auto-fecha, o que significa que se você quiser encerrar formalmente o elemento, você tem que adicionar uma barra "**/**" no final do próprio elemento e não uma tag de fechamento. No entanto, o tipo {{HTMLElement ("textarea")}} não é um elemento de auto-fechamento, então você tem que fechá-lo com a tag final adequada. Isso tem um impacto sobre um recurso específico de formulários HTML: a maneira como você define o valor padrão. Para definir o valor padrão de um elemento {{HTMLElement ("input")}} você tem que usar o atributo `value `como este:
```

File: files/pt-br/learn/forms/your_first_form/index.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+    </div>
+    <div class="button">
+        <button type="submit">Enviar sua mensagem</button>
+    </div>
+</form>
+```
+
+Um botão pode ser de três tipos: **`submit`**, **`reset`**, ou **`button`**.
+
+- Um clique sobre um botão de **`submit` **envia os dados do formulário para a página de web definida pelo atributo **`action `**do elemento {{HTMLElement ("form")}} .
```
Comment (josielrocha) [inline files/pt-br/learn/forms/your_first_form/index.md:123]:
```suggestion
- Um clique sobre um botão de **`submit`** envia os dados do formulário para a página de web definida pelo atributo **`action`** do elemento {{HTMLElement ("form")}} .
```

File: files/pt-br/learn/forms/your_first_form/index.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+## Enviar os dados para seu servidor web
+
+A última parte, e talvez a mais complicado, é lidar com dados de formulário no lado do servidor. Como dissemos antes, na maioria das vezes, um formulário HTML é uma forma conveniente para perguntar ao usuário os dados e enviá-lo para um servidor web.
+
+O elemento {{HTMLElement("form")}} definirá onde e como enviar os dados, graças ao atribudo _**action**_ e ao atributo _**method**_
+
+Mas não é o suficiente. Nós também precisamos dar um nome a nossos dados. Esses nomes são importantes em ambos os lados; no lado do navegador, ele informa ao navegador que nome dar a cada pedaço de dados, e no lado do servidor, ele permite que o servidor lidar com cada pedaço de dados pelo nome.
+
+Então, para nomear seus dados, você precisará usar o atributo \_**`name `**\_em cada campo do formulário que irá recolher uma parte específica dos dados:
```
Comment (josielrocha) [inline files/pt-br/learn/forms/your_first_form/index.md:242]:
```suggestion
Então, para nomear seus dados, você precisará usar o atributo _**`name`**_ em cada campo do formulário que irá recolher uma parte específica dos dados:
```

**Review by josielrocha — COMMENTED** [review]
Hi @SphinxKnight!

Thank you for your contribution.
I have found some issues, could you please fix them?

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Pull request merged**